Cargo.toml

```diff
@@ -40,3 +40,6 @@ unexpected_cfgs = { level = "warn", check-cfg = ["cfg(iroh_docsrs)", "cfg(iroh_l
 
 [workspace.lints.clippy]
 unused-async = "warn"
+
+[patch.crates-io]
+tokio-rustls-acme =  { git = "https://github.com/n0-computer/tokio-rustls-acme", branch = "feat-axum-8" }
```

Comment:
This is now merged. Probably needs a release & then we can depend on that?

Reply:
done